ctutil/ctutil.go

```diff
@@ -118,8 +118,46 @@ func VerifySCT(pubKey crypto.PublicKey, chain []*x509.Certificate, sct *ct.Signe
 		return fmt.Errorf("error creating signature verifier: %s", err)
 	}
 
-	entry := ct.LogEntry{Leaf: *leaf}
-	return s.VerifySCTSignature(*sct, entry)
+	return s.VerifySCTSignature(*sct, ct.LogEntry{Leaf: *leaf})
```

Comment:
Could replace some of VerifySCT() by delegating to VerifySCTWithVerifier().

Reply:
Done.